nautobot/core/ui/object_detail.py

```diff
@@ -732,6 +738,26 @@ def render_label(self, context):
             return bettertitle(context["object"]._meta.verbose_name)
         return super().render_label(context)
 
+    def render_value(self, key, value, context):
+        try:
+            field_instance = context["obj"]._meta.get_field(key)
+        except FieldDoesNotExist:
+            field_instance = None
+
+        if key == "_hierarchy":
+            return render_tree_queryset(value)
+
+        if isinstance(field_instance, URLField):
+            return hyperlinked_field(value, value)
+
+        if isinstance(field_instance, BooleanField):
+            return render_boolean(value)
+
+        if isinstance(field_instance, ManyToManyField):
+            return render_m2m(value.all())
```

Comment:
In *general* I don't think we should render M2M in `ObjectFieldsPanel` - that's what `ObjectTablePanel` is for. Content types *might* be an exception to that rule.

Reply:
Oooh I see, then illd limit it to only ContentType